src/backends/onnx/onnx_backend.cc

```diff
@@ -237,22 +241,37 @@ OnnxBackend::CreateExecutionContext(
 #endif  // TRTIS_ENABLE_GPU
   }
 
+  bool need_lock = false;
   if (Config().optimization().has_execution_accelerators()) {
     for (const auto& execution_accelerator : Config()
                                                  .optimization()
                                                  .execution_accelerators()
                                                  .cpu_execution_accelerator()) {
       if (execution_accelerator == kOpenVINOExecutionAccelerator) {
-        LOG_ERROR << "OpenVINO Execution Accelerator is not supported for "
-                  << instance_name;
+#ifdef TRTIS_ENABLE_ONNXRUNTIME_OPENVINO
+        need_lock = true;
+        RETURN_IF_ORT_ERROR(OrtSessionOptionsAppendExecutionProvider_OpenVINO(
+            session_options, "CPU"));
+        LOG_VERBOSE(1) << "OpenVINO Execution Accelerator is set for "
+                        << instance_name << " on device CPU"; 
+#else
+        LOG_ERROR << "OpenVINO Execution Accelerator is not enabled";
```

Comment:
Return an error status here. If model configuration is invalid in any way we want to fail.